src/DotNetLightning.Core/Serialize/Msgs/Msgs.fs

```diff
@@ -1281,7 +1281,15 @@ type ErrorMessage =
                 ls.WriteWithLen(this.Data)
 
         member this.GetFailureMsgData() =
-            System.Text.ASCIIEncoding.ASCII.GetString(this.Data)
+            let isPrintableAscii =
+                not <| Array.exists (fun asciiByte -> asciiByte < 32uy) this.Data
```

Comment:
magic number: 32uy; if you put it under a constant, and choose the right name, code will be more understandable